### DISCUSSION TITLE
Best way to Backup docker-compose Outline Selfhosted Installation

### DISCUSSION THREAD

**Discussion by victordariovera** [outline/outline#2103] [Self Hosting] [answered]
Hi! What is the best way to backup my content in a selfhosted installation of Outline in a automatic and scheduled way? I can use Outline API?

**Comment by tommoor** [comment, marked as answer]
I would recommend to use a Postgres read-replica, or schedule pgdump at an interval that makes sense for your team.

The API has good output options but doesn't allow reimporting to the same fidelity that a db backup does.

**Comment by victordariovera** [comment]
Thanks, I will investigate how to backup in that way. I want to backup in
plain markdown files with images too as a future-proof and agnostic backup
format. I love Outline, but can be interesting to have plain markdown files
in a disaster scenario.

**Reply by tommoor** [reply]
Well, the database contains the markdown files in a single table, so it's much the same thing! But you can also use the `collections.export` or `documents.export` endpoints to output plain txt files in Markdown format
